pytest-selenium/pages/osweb.py

```diff
@@ -116,3 +124,30 @@ def osweb_username(self, element):
         """Get the username of the logged in user."""
         element1 = self.username(element)
         return " ".join(element1.split()[:2])
+
+    def notification_dialog_displayed(self) -> bool:
+        """Return True if the dialog box is displayed.
+        :return: ``True`` if the dialog box is displayed
+        :rtype: bool
+        """
+        try:
+            return bool(self.find_element(*self._dialog_locator))
+        except NoSuchElementException:
+            return False
+
+    def click_notification_got_it(self):
+        """Click the 'Got it!' button.
+        :return: the home page
+        :rtype: :py:class:`~pages.web.home.WebHome`
+        """
+        button = self.find_element(*self._got_it_button_locator)
+        Utilities.click_option(self.driver, element=button)
+        self.wait.until(lambda _: not self.notification_dialog_displayed())
```

Comment:
With the previous comment, this becomes:
`lambda _: not self.notification_dialog_displayed`